CLANG64 got broken

### Platform

Windows

### Compiler and build tool versions

MSYS2 CLANG64, Ninja

### Operating system version

Windows 11

### Build commands used

mkdir build
cd build
cmake ..
cmake --build .

### What happens

```
FAILED: [code=1] CMakeFiles/Common.dir/Common/Render/Text/draw_text_uwp.cpp.obj
ccache C:\msys64\clang64\bin\ccache.exe C:\msys64\clang64\bin\c++.exe -DASSETS_DIR="\"C:/Program Files (x86)/PPSSPP/share/ppsspp/assets/\"" -DMINIUPNP_EXPORTS -DMINIUPNP_STATICLIB -DPNG_ARM_NEON_OPT=0 -DSHARED_ZLIB -DSTACK_LINE_READER_BUFFER_SIZE=1024 -DUNICODE -DUSE_DISCORD=1 -DUSE_SDL3_TTF -DUSE_SDL3_TTF_FONTCONFIG -DUSING_WIN_UI -DWIN32 -DWITH_UPNP -DZ7_CRC_NUM_TABLES=1 -D_BSD_SOURCE -D_DEFAULT_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE=1 -D_UNICODE -D_XOPEN_SOURCE=700 -D_XOPEN_SOURCE_EXTENDED -D__BSD_VISIBLE=1 -D__STDC_CONSTANT_MACROS -IC:/msys64/home/Cal/ppsspp/ext/glslang -IC:/msys64/home/Cal/ppsspp/ext/glew -IC:/msys64/home/Cal/ppsspp -IC:/msys64/home/Cal/ppsspp/ext/OpenXR-SDK/include -IC:/msys64/home/Cal/ppsspp/ext -IC:/msys64/home/Cal/ppsspp/Common -IC:/msys64/home/Cal/ppsspp/ext/libzip -IC:/msys64/home/Cal/ppsspp/ext/lzma-sdk -IC:/msys64/home/Cal/ppsspp/ext/zstd/lib -IC:/msys64/home/Cal/ppsspp/ext/libchdr/include -IC:/msys64/home/Cal/ppsspp/build -IC:/msys64/home/Cal/ppsspp/ext/miniupnp/miniupnpc/src -IC:/msys64/home/Cal/ppsspp/ext/miniupnp/miniupnpc/include -IC:/msys64/home/Cal/ppsspp/ext/snappy/. -IC:/msys64/home/Cal/ppsspp/ext/cmake/cpu_features/../../cpu_features/include -O3 -DNDEBUG -std=gnu++17 -Qunused-arguments -fcolor-diagnostics -Wall -Werror=return-type -Wno-unused-function -Wno-sign-compare -Wno-unused-but-set-variable -Wno-reorder -Wno-unknown-pragmas -Wno-unused-value -Wno-unused-variable -Wno-error=incompatible-pointer-types -Wno-deprecated-declarations -Wno-missing-braces -Wno-nullability-completeness -Wno-tautological-pointer-compare -Wno-deprecated-register -Wno-sign-conversion -Wno-shorten-64-to-32 -Wformat -Wno-multichar -fno-strict-aliasing -fno-math-errno -msse2 -MD -MT CMakeFiles/Common.dir/Common/Render/Text/draw_text_uwp.cpp.obj -MF CMakeFiles\Common.dir\Common\Render\Text\draw_text_uwp.cpp.obj.d -o CMakeFiles/Common.dir/Common/Render/Text/draw_text_uwp.cpp.obj -c C:/msys64/home/Cal/ppsspp/Common/Render/Text/draw_text_uwp.cpp
C:/msys64/home/Cal/ppsspp/Common/Render/Text/draw_text_uwp.cpp:180:38: error: cannot initialize a parameter of type 'IDWriteFontFamily1 **' (aka 'struct IDWriteFontFamily1 **') with an rvalue of type 'IDWriteFontFamily **' (aka 'struct IDWriteFontFamily **')
  180 |                 m_fontCollection->GetFontFamily(i, &family);
      |                                                    ^~~~~~~
C:/msys64/clang64/include/dwrite_3.h:2528:30: note: passing argument to parameter 'family' here
 2528 |         IDWriteFontFamily1 **family) = 0;
      |                              ^

```


### PPSSPP version affected

v1.20.4-413-g676724ee5e

### Last working version

v1.20.3-287-g53bbe11f30 (may work with new commits, just old one i tested)

### Checklist

- [x] Test in the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already fixed.
- [x] Make sure to run `git submodule update --init --recursive` before building.
- [x] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [x] Try deleting the build directory and running the build again.
- [ ] Check GitHub Actions, which builds every merge and PR.
- [ ] Include logs and help us reproduce.

--- a/Common/Render/Text/draw_text_uwp.cpp
+++ b/Common/Render/Text/draw_text_uwp.cpp
@@ -176,7 +176,7 @@ TextDrawerUWP::TextDrawerUWP(Draw::DrawContext *draw) : TextDrawer(draw), ctx_(n
 
 	UINT32 familyCount = m_fontCollection->GetFontFamilyCount();
 	for (UINT32 i = 0; i < familyCount; ++i) {
-		IDWriteFontFamily *family;
+		IDWriteFontFamily1 *family;
 		m_fontCollection->GetFontFamily(i, &family);
 
 		IDWriteLocalizedStrings *names;

